server/bundles/io.cloudbeaver.service.data.transfer/src/io/cloudbeaver/service/data/transfer/impl/WebServiceDataTransfer.java

```diff
@@ -282,4 +327,41 @@ public void fetchRow(@NotNull DBCSession session, @NotNull DBCResultSet resultSe
         consumer.finishTransfer(monitor, false);
     }
 
+    private void importData(
+            DBRProgressMonitor monitor,
+            DataTransferProcessorDescriptor processor,
+            @NotNull DBSDataManipulator dataContainer,
+            Path path) throws DBException
+    {
+        IDataTransferProcessor processorInstance = processor.getInstance();
+
+        if (dataContainer.getDataSource() != null) {
+            DatabaseTransferConsumer consumer = new DatabaseTransferConsumer(dataContainer);
+
+            DatabaseConsumerSettings databaseConsumerSettings = new DatabaseConsumerSettings();
+            databaseConsumerSettings.setContainer((DBSObjectContainer) dataContainer.getDataSource());
+            consumer.setSettings(databaseConsumerSettings);
+            DatabaseMappingContainer databaseMappingContainer = new DatabaseMappingContainer(databaseConsumerSettings, dataContainer);
+            databaseMappingContainer.getAttributeMappings(monitor);
+            databaseMappingContainer.setTarget(dataContainer);
+            consumer.setContainerMapping(databaseMappingContainer);
```

Comment:
initTransfer ?

Reply:
It doesnt work, we dont have transfer and bad consumer in export